wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,11 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'fill-200x200|height-150')
```

Comment:
`render_image_tag` here is constructing an `{% image %}` tag with the given parameter, so I would expect the filters here to be space-separated, not pipe separated. The fact that the test is apparently succeeding anyhow suggests that something fishy is going on with these tests...